src/main/processor/download-task.processor.ts

```diff
@@ -23,7 +23,7 @@ export class DownloadTaskProcessor implements TaskProcessor<DownloadTaskPayload>
   constructor(private readonly configService: ConfigService<MainConfig>) {}
 
   async process(task: AssetTask<DownloadTaskPayload>): Promise<void> {
-    this.logger.debug(`[${task.id}] - Processing download task`, task.payload)
+    this.logger.debug(`[${task.id}] - Processing download task ${JSON.stringify(task.payload)}`)
```

Comment:
This is a sign to me that this logger isn't very good. I mentioned [Pino](https://www.npmjs.com/package/nestjs-pino/v/0.3.1) has a nest version so we can pass objects and errors in directly and it will format it properly. Btw there is a pretty package to make it super readable.

Reply:
Can we avoid holding up this MR for that?